### PR TITLE
Clarify use of admin credentials in install

### PR DESCRIPTION
**Pull request by andrewtchin**

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 1]
```diff
@@ -87,10 +87,10 @@ You can deploy multiple vSphere Integrated Containers appliances to the same vCe
 
 12. Enter the connection details for the vCenter Server instance on which you deployed the appliance.
 
-     - The address and single sign-on credentials of vCenter Server.
+     - The address and single sign-on administrator credentials of vCenter Server.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: The vCenter Server address and the Single Sign-on credentials for a vSphere administrator account.

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 2]
```diff
@@ -87,10 +87,10 @@ You can deploy multiple vSphere Integrated Containers appliances to the same vCe
...
      - If vCenter Server is managed by an external Platform Services Controller, enter the FQDN and administrator domain for the Platform Services Controller. If vCenter Server is managed by an embedded Platform Services Controller, leave the External PSC text boxes empty.
 
-    **IMPORTANT**: The installation process requires the single sign-on credentials to register vSphere Integrated Containers Management Portal and Registry with the Platform Services Controller and to tag the appliance VM for use in Docker content trust. The vSphere Integrated Containers Management Portal and Registry services cannot start if you do not complete this step.
+    **IMPORTANT**: The installation process requires the single sign-on administrator credentials to register vSphere Integrated Containers Management Portal and Registry with the Platform Services Controller and to tag the appliance VM for use in Docker content trust. Administrator credentials are not stored on the appliance VM after use in the installation process. The vSphere Integrated Containers Management Portal and Registry services cannot start if you do not complete this step.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: The installation process requires administrator credentials to register vSphere Integrated Containers ....

**Comment by stuclem** [comment]
@andrewtchin looking at this topic again, we don't mention that you need to use a vSphere admin account to deploy the OVA. Is this the case?

**Comment by andrewtchin** [comment]
@stuclem I think any account that has permissions to deploy an OVA should work, though I've never tried it with anything other than admin.

**Comment by stuclem** [comment]
OK, thanks @andrewtchin. In that case we're good to merge this one.

**Pull request merged**